ApeWorX/ape
# feat: eth-tester provider

### What I did

Make the eth-tester provider able to be used in `ape test`.

### How I did it

* Use the latest web3.py (released today!) that has the eth-tester mnemonic related changes
* Call the `from_mnemonic()` method
* Handle `TransactionFailed` as the eth-tester form of `ContractLogicError`.

### How to verify it

Run tests using eth-tester!
Try running scripts too, if you dare!
See https://github.com/unparalleled-js/ape-demo-project for example scripts / tests if that is helpful.

### Checklist
